src/components/ui/layouts/CraftingRequirements.tsx

```diff
@@ -82,7 +82,7 @@ export const CraftingRequirements: React.FC<Props> = ({
   stock,
   isLimitedItem = false,
   details,
-  boosts = [],
+  boost: boost,
```

Comment:
Is this necessary? Can it just be `boost,`?

Reply:
whoops, forgot to remove this while renaming variables.  Fixed.